src/styles/components/_about-us-introduction.scss

```diff
@@ -0,0 +1,95 @@
+.about-us-introduction {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: rem(250px);
+
+  @include respond-above('sm') {
+    height: rem(400px);
+  }
+  @include respond-above('md') {
+    height: rem(500px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(600px);
+  }
+  @include respond-above('xl') {
+    height: rem(700px);
+  }
+
+  &__title {
+    width: 58%;
+    height: 12.5%;
+    margin-block-start: 0%;
+
+    color: $dark;
+    -webkit-text-stroke: rem(1px) $dark;
+
+    @include respond-above('md') {
+      font-size: rem(28px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(32px);
+    }
+
+    @include respond-above('xl') {
+      width: 45%;
+
+      font-size: rem(36px);
+    }
+
+    font-size: rem(24px);
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+  }
+
+  &__description {
```

Comment:
Please remove the unused `margin`s.
![Screen Shot 2564-02-26 at 01 27 58](https://user-images.githubusercontent.com/32289827/109200490-4424cf80-77d3-11eb-985d-d7c840bff51e.png)